src/resources/js/fileFactories/Laravel/pipes/MigrationPipe.js

```diff
@@ -59,18 +59,32 @@ export default class MigrationPipe extends BasePipe {
         if(attribute.nullable || attribute.dataType === "timestamp") chainings += "->nullable()";
         if(attribute.unique) chainings += "->unique()";
         return chainings
-
     }
 
-    migrationTimeStamp(index) {
+    _migrationTimeStamp(index) {
```

Comment:
Wait, this became unreachable/unused code right? We can just remove it.

Reply:
Thanks just fixed it!